onnxruntime/python/tools/transformers/models/whisper/whisper_encoder_decoder_init.py

```diff
@@ -120,7 +132,8 @@ def export_onnx(
         )
         input_list = inputs.to_list()
 
-        out = model(inputs.encoder_input_ids, inputs.decoder_input_ids)
+        cloned_model = copy.deepcopy(model).to(device)
```

Comment:
Why is a copy of `model` needed?

Reply:
@shubhambhokare1 did you see this one?

Reply:
Conversion process seems to be erroring out without model copying